src/services/deployments.service.ts

```diff
@@ -94,4 +95,18 @@ export class DeploymentsService {
 			return { data: undefined, error };
 		}
 	}
+
+	static async delete(deploymentId: string): Promise<ServiceResponse<undefined>> {
+		try {
+			await deploymentsClient.delete({ deploymentId });
+			return { data: undefined, error: undefined };
+		} catch (error) {
+			const errorMessage = `${translate().t("deployments.deleteFailedId", {
+				deploymentId,
+			})} - ${(error as Error).message}`;
```

Comment:
print to log the expanded error message, and for the notification only the "title" and only the short description.